FreeBSD/usr/ports/net/ceph-devel/distinfo

```diff
@@ -0,0 +1,35 @@
+TIMESTAMP = 1486914967
```

Comment:
@wjwithagen i don't think we should have the port files in upstream repo as it includes a point-in-time meta info like `distinfo`. you can have this in your own dedicated repo for maintaining the port before it's accepted[1] by FreeBSD ports. and it's not advisable to keep the packaging in upstream unless we are able to test it using our test infrastructure.

---
[1] https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=217046